pkg/tls/ca.go

```diff
@@ -157,7 +157,7 @@ func GenerateRootCAWithDefaults(name string) (*CA, error) {
 	return CreateRootCA(name, key, Validity{})
 }
 
-// GenerateCA generates a new intermdiary CA.
+// GenerateCA generates a new intermediate CA.
```

Comment:
I think "intermediary" was intentional, but "intermediate" seems to be a more common term for this.  :+1:

Reply:
Yeah, that's exactly the thought process I took.